ui/shared/AccountSelector.qml

```diff
@@ -22,10 +20,38 @@ Item {
     property string showAssetBalance: ""
     property int dropdownWidth: width
     property alias dropdownAlignment: select.menuAlignment
+    property bool isValid: true
+    property var reset: function() {}
+
+    function resetInternal() {
+        accounts = undefined
+        selectedAccount = undefined
+        isValid = true
+    }
+
+    onSelectedAccountChanged: {
+        if (selectedAccount) {
```

Comment:
Can we get a good old reverse condition and early exit :)

Reply:
Done.